x/incentive/params.go

```diff
@@ -107,22 +108,18 @@ func validateMaxUnbondings(i interface{}) error {
 }
 
 func validateCommunityFundAddress(i interface{}) error {
-	_, ok := i.(string)
+	s, ok := i.(string)
```

Comment:
Why this function takes interface, rather than a string? Please don't use deprecated params module.

Reply:
I'll update that - thanks for catching the old pattern carrying over into new code